propagator/google_cloud_platform/lib/opentelemetry/propagator/google_cloud_platform.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'opentelemetry-api'
```

Comment:
Is this `require` necessary if it's also required in `lib/opentelemetry-propagator-google_cloud_platform.rb`?